### PR TITLE
stabilized needs_drop (fixes #41890)

### LINKED ISSUE #41890: Tracking Issue for needs_drop
About to submit a PR for exposing intrinsics::needs_drop under mem. Needs a tracking issue.

Here it is!

### PR DESCRIPTION
**Pull request by budziq**
fixes #41890

### CODE REVIEW

File: src/libcore/mem.rs [hunk 1]
```diff
@@ -359,7 +358,7 @@ pub fn align_of_val<T: ?Sized>(val: &T) -> usize {
 /// }
 /// ```
 #[inline]
-#[unstable(feature = "needs_drop", issue = "41890")]
+#[stable(feature = "needs_drop", since = "1.22.0")]
```
Comment (oherrala) [inline src/libcore/mem.rs]: If this was released with Rust 1.21, is the since field wrong?

Comment (kennytm) [reply src/libcore/mem.rs]: @oherrala The versions has been fixed in 874124b2c73dc764aa83f7ba9711a4170c0aeffc 🙂

Comment (oherrala) [reply src/libcore/mem.rs]: @kennytm, ah, cool. Sorry about the noise.

**Pull request merged**